new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Docker.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl.primitive;
+
+
+public class Docker {
+
+  /**
+   * Log in to a Docker registry.
+   * @param registryName registry name
+   * @param username user
+   * @param password password
+   * @return true if successfull
+   */
+  public static boolean login(String registryName, String username, String password) {
+    String cmdToExecute = String.format("docker login %s -u %s -p \"%s\"",
+        registryName, username, password);
+    return new Command()
+        .withParams(new CommandParams()
+            .command(cmdToExecute))
+        .execute();
+  }
+
+  /**
+   * Push an image to a registry
+   * @param image image
```

Comment:
what does image mean? is the fully qualified docker image name, including the repository and the tag?

Reply:
yes, changing the param description to `fully qualified docker image, image name:image tag`